app/assets/javascripts/paypal_account_settings.js

```diff
@@ -2,23 +2,28 @@ window.ST = window.ST || {};
 
 (function(module) {
 
-  module.initializePayPalAccountForm = function(formId) {
-    var form = $('#'+formId);
+
+  module.initializeNewPayPalAccountHandler = function(button_id, action) {
```

Comment:
Btw. in the code we have always used `Paypal`, which is wrong, yes, but for the sake of consistency, should we use `Paypal` also here?